src/Pact/Gas/Table.hs

```diff
@@ -68,6 +68,7 @@ defaultGasTable =
    ,("identity",             const 2)
    ,("if",                   const 1)
    ,("int-to-str",           const 1)
+   ,("is-charset",           const 1) -- TODO benchmark
```

Comment:
Should we also add all of the un-benchmarked "natives" (the constants and deprecated ones) as well?
i.e.    
```
-- Constants and deprecated functions
   ,("CHARSET_ASCII",       const 1)
   ,("CHARSET_LATIN1",      const 1)
   ,("verify-spv",          const 1)
   ,("public-chain-data",   const 1)
   ,("list",                const 1)
```
I add them to the gas table in my benchmark PR: https://github.com/kadena-io/pact/blob/5f7203685091305bc2cd8fed77effe9d50affa6b/src/Pact/Gas/Table.hs#L177

Reply:
Good idea, just in case